File: readcsv_collect.jl
using CSV, DataFrames
df = CSV.read("data_with_classes.csv",DataFrame)
# println(df.Class)
global matclass1=Array{Any}(undef, 0)
global matclass2=Array{Any}(undef, 0)
global matclass3=Array{Any}(undef, 0)

for row in eachrow(df)
    global matclass1,matclass2,matclass3
    data = row.Class
    if data == 1 
        push!(matclass1,row[1:5])
    end
    if data == 2 
        push!(matclass2,row[1:5])
    end
    if data == 3
        push!(matclass3,row[1:5])
    end
end

println(matclass1)
println(matclass2)
println(matclass3) 

File: readcsv_collect.py
import csv

buf=[]
matclass1=[]
matclass2=[]
matclass3=[]
with open('data_with_classes.csv', 'r', newline='') as df:
    rows = csv.reader(df, delimiter=',')
    for row in rows:
        buf.append(row)

for data in buf:
    if data[5] == '1':
        matclass1.append(data[0:5])
    if data[5] == '2':
        matclass2.append(data[0:5])
    if data[5] == '3':
        matclass3.append(data[0:5])
    